### ISSUE TITLE
how to configure to select some countries only?

### ISSUE THREAD

**Issue by david-young-fax** [mojoaxel/bootstrap-select-country#4]
it's good to show all countries. But in our use case scenario, we don't support some countries for our application.

it's quite confusing if we just show them all. Is it possible for us to configure and show desired countries?

**Comment by mojoaxel** [comment]
This feature is not implemented yet.

**Comment by mojoaxel** [comment]
I just [released version 4.1.0](https://github.com/mojoaxel/bootstrap-select-country/releases/tag/v4.1.0). You now can define a list of countries like so:
```html
<select class="selectpicker countrypicker" data-countries="AT,BE,BG,HR" ></select>
```
